app/src/main/java/com/keylesspalace/tusky/components/compose/ComposeActivity.kt

```diff
@@ -682,7 +682,17 @@ class ComposeActivity : BaseActivity(),
     }
 
     private fun updateVisibleCharactersLeft() {
-        composeCharactersLeftView.text = String.format(Locale.getDefault(), "%d", maximumTootCharacters - calculateTextLength())
+        var remainingLength = maximumTootCharacters - calculateTextLength();
+        composeCharactersLeftView.text = String.format(Locale.getDefault(), "%d", remainingLength)
+
+        if(remainingLength < 0) {
+            val textColor = ThemeUtils.getColor(this, android.R.attr.colorError)
```

Comment:
Field requires API level 26 (current min is 21): `android.R.attr#colorError` - better define your own color, ideally one for the dark themes and one for the light theme